### PR TITLE
reports: add week 25 dev report

### PR DESCRIPTION
**Pull request by avsm**

### CODE REVIEW

File: reports/2017-06-25.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Weekly LinuxKit dev report for 2017-06-19 to 2017-06-25 (week 25)
+
+This report covers weekly developments in the [linuxkit], [linuxkit-ci] and [virtsock] repositories. The Moby Summit held in San Francisco was very active and well attended ([blog](http://mobyproject.org/blog/2017/06/26/moby-summit-recap/) / [notes](2017-06-19-summit.md) [#2090] [#2033] [@mgoelzer] [@justincormack]). 
+
+* **Kernel GPG verification:** The kernel compilation containers now verify the GPG and SHA256
+  checksums before building the binaries. ([#2062] [#2083] [@mscribe] [@justincormack] [@rn] [@riyazdf]).
+  The base Alpine build image now includs `gnupg` to support this feature ([#2091] [@riyazdf] [@rn]).
```
Comment (justincormack) [inline reports/2017-06-25.md]: "includes"

**Comment by justincormack** [comment]
minor typo but LGTM otherwise

File: reports/2017-06-25.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+* **Security SIG on Landlock:**  The second Moby Security SIG focussed on the [Landlock](https://github.com/landlock-lsm) security module that provides unprivileged fine-grained sandboxing to applications.  There are [meeting notes](2017-06-21.md) as well as videos and forum links ([#2087] [#2089] [#2073] [@riyazdf]).
```
Comment (riyazdf) [inline reports/2017-06-25.md]: s/second/third

File: reports/2017-06-25.md (hunk 1 above)
Comment (avsm) [inline reports/2017-06-25.md]: fixed in 3d45046bbe71fd10b46256dd8bd62eb24f0d2fb7

File: reports/2017-06-25.md (hunk 2 above)
Comment (avsm) [inline reports/2017-06-25.md]: fixed in 3d45046bbe71fd10b46256dd8bd62eb24f0d2fb7

**Pull request merged**